control-plane/roles/monitoring/templates/metal-metrics-exporter/metal-metrics-exporter-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: metal-metrics-exporter-config
+type: Opaque
+stringData:
+  url: http://metal-api.metal-control-plane.svc:8080/metal
+  hmac: metal-admin
```

Comment:
Needs to be put into role variables.

Reply:
https://github.com/metal-stack/metal-roles/commit/50bf25e3659cd4e8be5d311cd3796939de3295a3